README.md

```diff
@@ -1,7 +1,88 @@
 # Kinesis Flink App Hudi Benchmark
 
-AWS Kinesis Flink App processing a real time streaming input that writes the output in different file formats to S3
+## Contributors
+
+This repository has been developed primarily by [@adrij](https://github.com/adrijh) and [@alfonjerezi](https://github.com/alfonjerezi).
+
+## Introduction
+
+This project deploys an architecture in AWS which ingest and processes streaming data with Kinesis Flink Application and writes the output to S3 in Hudi and JSON format.
 
 ## Architecture
 
 ![Alt text](images/flink-hudi.png?raw=true "Architecture")
+
+## Documentation
+
+Articles:
+ - First article: LakeHouse Flink streaming (Pending)
+
+## Requirements
+
+- You must own an AWS account and have an Access Key to be able to authenticate. You must create a profile called "practica_cloud" and put the previous credentials in it. See [here](https://docs.aws.amazon.com/cli/latest/reference/configure/) steps to create it.
```

Comment:
What do you think about suggesting to change the profile to their own instead? Feels weird to force someone following the repo to create a profile called "practica_cloud".

Maybe even removing the profile from backend.tf and provider.tf so it works with the default profile

Reply:
Uhm I left it for the user to fill in wherever we left the value <YOUR_AWS_PROFILE>, I prefer putting it in this way so its explicit, beginners in the cloud might have a hard time understanding we are using the default credentials and where they are coming from. Its not ideal because they have to change it in 5 files but terraform doesn't allow variables in the provider and backend configurations.

Let me know what you think.